libraries/AP_ServoRelayEvents/AP_ServoRelayEvents.cpp

```diff
@@ -44,6 +44,14 @@ bool AP_ServoRelayEvents::do_set_servo(uint8_t _channel, uint16_t pwm)
     case SRV_Channel::k_gripper:
     case SRV_Channel::k_rcin1 ... SRV_Channel::k_rcin16: // rc pass-thru
         break;
+    case SRV_Channel::k_rcin1_mapped ... SRV_Channel::k_rcin16_mapped: {
+        // mapped channels are set up with a -/+ 4500 angle range by
+        // SRV_Channel::aux_servo_function_setup
+        int16_t angle_scaled = constrain_uint16(pwm, 1000, 2000);
+        angle_scaled = (angle_scaled - 1500) * 9; // 1000 ... 2000 -> -500 ... 500 -> -4500 ... 4500
```

Comment:
```suggestion
        angle_scaled = (angle_scaled - 1500) * (4500/500); // 1000 ... 2000 -> -500 ... 500 -> -4500 ... 4500
```
The compiler will evaluate it to 9 in any case and I think this makes it a little easier to understand I think.